docs/UserGuide.md

```diff
@@ -146,6 +146,44 @@ Examples:
 * `changeqty 1 a/+1` increases the quantity of the 1st food item by 1.
 * `changeqty 2 a/-2` decreases the quantity of the 2nd food item by 2.
 
+### Sort food items by description: `sortdesc`
+
+Sorts the list of food items by description, then by expiry date from oldest to newest, then by priority from high to low. 
+
+* Sorting by description consists of first sorting lexicographically, then by the description's first characters.
+* For the same letters, descriptions with upper case first characters will be ordered lower than descriptions with lower case first characters. 
```

Comment:
I feel this point is a bit confusing. Will this be better "If two items have the same description content but difference in the case, the one with upper case will be lower than the one with the lower case"?

Reply:
Items of different description content still consider the case of the first characters though. Perhaps we can discuss a better way to represent the meaning accurately and clearly during our next meeting